hw1/02-palindrome.html

```diff
@@ -1,27 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>02 - Palindrome</title>
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
-    <script defer src="02-palindrome.js"></script>
-  </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
-        </div>
-      </section>
-    </main>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>02 - Palindrome</title>
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+		/>
+		<script defer src="02-palindrome.js"></script>
+	</head>
+	<body class="bg-primary">
+		<main class="w-50 mx-auto my-5">
+			<section class="bg-light rounded p-5">
+				<h1>Exercise 02 - Palindrome</h1>
+				<div>
+					<label>Enter a positive number:</label>
```

Comment:
Since the label is not associated with the input element, this might probably cause error in the WAVE analysis. Can you verify it once using WAVE from your end.

Reply:
Thanks for noticing. I have added for to the label now.